cirq-core/cirq/optimizers/eject_phased_paulis.py

```diff
@@ -63,52 +64,53 @@ def optimize_circuit(self, circuit: circuits.Circuit):
 
         for moment_index, moment in enumerate(circuit):
             for op in moment.operations:
-                affected = [q for q in op.qubits if q in state.held_w_phases]
+                if NoCompileTag not in op.tags:
```

Comment:
I would probably be consistent with above and say

```
if NoCompileTag in op.tags:
   continue
```

This avoids the negative condition and makes the change cleaner since all indentation doesn't change.